packages/core/src/common/tetherUtils.ts

```diff
@@ -14,6 +14,19 @@ const DEFAULT_CONSTRAINTS = {
     to: "scrollParent",
 };
 
+// per https://github.com/HubSpot/tether/pull/204,
```

Comment:
minor: I think the line breaks are a little too aggressive here -- you could comfortably take up more horizontal space